src/component/veilederverktoy/tildel-veileder/tildel-veileder-kvittering.tsx

```diff
@@ -17,7 +21,14 @@ export function TildelVeilederKvittering(props: TildelVeilederKvitteringProps) {
             <BodyShort size="small">
                 Du har tildelt veileder {props.tildeltVeilederNavn}. Det kan ta noe tid før brukeren er i Min oversikt.
             </BodyShort>
-            <Button variant="primary" size="small" onClick={hideModal}>
+            <Button
+                variant="primary"
+                size="small"
+                onClick={() => {
+                    setTimeout(() => setUfordeltbruker(false), 1000);
```

Comment:
i setUfordeltbruker(false) her fungerer egentlig slik at man oppdaterer det som vises i påvente av svar fra backend, men fordi frontenden da gjør kallet før kafkamelding og alt av operasjoner i backend blir ferdig må vi vente litt med å gjøre kallet før vi kan be om at man henter det på nytt 👍